hoverkite-firmware/src/protocol.rs

```diff
@@ -12,135 +11,93 @@ use gd32f1x0_hal::{
     prelude::*,
     serial::{Rx, Tx},
 };
-use hoverkite_protocol::{Command, DirectedCommand, Side};
+#[allow(unused_imports)]
+use hoverkite_protocol::{Command, DirectedCommand, ProtocolError, Response, Side, SideResponse};
+#[allow(unused_imports)]
+use nb::Error::{Other, WouldBlock};
 
 #[macro_export]
 macro_rules! log {
     ($dst:expr, $($arg:tt)*) => (
 		{
-			core::fmt::Write::write_char(&mut $dst, '"').unwrap();
-			core::fmt::Write::write_fmt(&mut $dst, format_args!($($arg)*)).unwrap();
-			core::fmt::Write::write_char(&mut $dst,'\n').unwrap();
+            SideResponse {
```

Comment:
Don't you need to fully-qualify `SideResponse` too, so that it works wherever the macro is used?